charts/rd-professional-api/values.preview.template.yaml

```diff
@@ -7,6 +7,7 @@ java:
     POSTGRES_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
     POSTGRES_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     LAUNCH_DARKLY_ENV: "preview"
+    USER_PROFILE_URL: http://rd-user-profile-api-pr-581.service.core-compute-preview.internal
```

Comment:
need to be removed?

Reply:
Yes, can be removed now that testing is completed

Reply:
Change completed here: https://github.com/hmcts/rd-professional-api/pull/954/commits/b142797a575f44d1b8227e9976a872b68c97e4aa